src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,9 @@
+export const Tab = ({ tab, findActive, active }) => {
```

Comment:
The prop `findActive` should be renamed to `onTabSelected` and `active` should be renamed to `activeTabId` to match the task description. This will ensure that the `Tab` component receives the correct props as intended.